plugin/build.gradle.kts

```diff
@@ -24,7 +25,9 @@ kotlin {
     abiValidation {
         enabled = true
     }
-    compilerOptions.freeCompilerArgs.add("-Xjvm-default=all-compatibility")
+    compilerOptions {
+        jvmDefault = JvmDefaultMode.NO_COMPATIBILITY
+    }
```

Comment:
Regardless the DSL change, can you elaborate the impact of this? I remember adding it to let Kotlin's default method on interfaces to behave (and be Groovy compatible) with Java ones.

Reply:
After reading https://kotlinlang.org/docs/whatsnew22.html#changes-to-default-method-generation-for-interface-functions I'm wondering what should be the right configuration to set for this, considering the "Groovy compatibility" use case. Seems either `enable` and `no-compatibility` will work 🤔 . Any thoughts on that?